daemon/execdriver/native/apparmor.go

```diff
@@ -76,7 +73,6 @@ profile docker-unconfined flags=(attach_disconnected,mediate_deleted,complain) {
   change_profile -> *,
 
   ptrace,
-  signal,
```

Comment:
Although this fixes 2.7, the problem is that users of AppArmor 2.8+ will need the signal lines in order to do a 'docker kill' with docker-in-docker. In order not to break them, we'd need to remove the use of 'docker-unconfined' in apparmor.go.

It's okay to keep the profile in here, users would just have to select it manually (those needing to workaround problems with /usr/bin/tcpdump and similar binaries).